### PR TITLE
Move more encryption endpoints off master

### PR DESCRIPTION
**Pull request by erikjohnston**
These are `/keys/claim` and `/room_keys/*` APIs.

### CODE REVIEW

File: changelog.d/9068.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Add experimental support for handling `/keys/claim` and `/room_keys` APIs on worker processes.
```
Comment (clokep) [inline changelog.d/9068.feature:1]: Should this be combined with the one from #9044?

**Review by clokep — APPROVED** [review]
Seems fine! I haven't reasoned too much about each of the additional methods now available to workers.

File: changelog.d/9068.feature (hunk 1 above)
Comment (erikjohnston) [reply changelog.d/9068.feature:1]: Err, don't know really. I don't know whether its better to have the separate or combine them (in terms of readability).

(I'm going to leave it for now to let the tests run)

**Comment by erikjohnston** [comment]
Rebased on top of what we've deployed to matrix.org

**Pull request merged**